insights/client/__init__.py

```diff
@@ -667,11 +667,9 @@ def format_config(config):
     # Log config except the password
     # and proxy as it might have a pw as well
     config_copy = config.copy()
-    try:
-        del config_copy["password"]
-        del config_copy["proxy"]
-    finally:
-        return json.dumps(config_copy, indent=4)
+    config_copy.pop("password", None)
+    config_copy.pop("proxy", None)
+    return json.dumps(config_copy, indent=4)
```

Comment:
Should I remove this function? `format_config()` is never called, imported or referenced in entire repository of insights-core and insights-client

@m-horky what do you think? I would remove it, however I don't want to break anything unexpectedly